README.md

```diff
@@ -2,11 +2,17 @@
 
 # How can a developer help The New York Times newsroom?
 
+## We Are Currently Hiring!
+
+  * [Newsroom UX Software Engineer](https://nytimes.wd5.myworkdayjobs.com/en-US/News/job/New-York-NY/Newsroom-UX-Software-Engineer_REQ-004251-3)
+  * [Full-Stack Newsroom Software Engineer](#) Link TK soon!
```

Comment:
I will update this with the URL once we have it, whether in `master` or if we still have this outstanding PR.